.github/workflows/cypress-tests.yml

```diff
@@ -0,0 +1,39 @@
+name: "dhis2: cypress tests"
+
+on: [push]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        containers: [1, 2, 3]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          record: true
+          parallel: true
+          group: e2e-chrome-parallel
+          browser: chrome
+          start: 'yarn start'
+          wait-on: 'http://localhost:3000'
+          # wait for 200 secs for the server to respond
+          wait-on-timeout: 200
+
+      - name: Upload artifacts
+        if: failure()
+        uses: actions/upload-artifact@v1
+        with:
+          name: failed-screenshots
+          path: cypress/screenshots
+    env:
+      CI: true
+      CYPRESS_RECORD_KEY: '6b0bce0d-a4e8-417b-bbee-9157cbe9a999'
```

Comment:
cypress provides us with a dashboard view. there you can see failing tests etc etc. this key is given by them and is not a "secret" therefore I am adding it here.